.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules
 client/dist/bundle-images-service.js
-database/config.js
\ No newline at end of file
+database/config.js
+package-lock.json
```

Comment:
I think you're supposed to keep this

Reply:
Right you are. I thought I had heard different, but the internet tells me it should not be ignored